.env.example

```
@@ -59,10 +59,10 @@ NEXT_PUBLIC_PLAYWRIGHT_TEST=false
 # --------------------------------------------
 # Shared Frontend
 # --------------------------------------------
-NEXT_PUBLIC_API_ENDPOINT=http://genfeed.localhost:3010/v1
-NEXT_PUBLIC_MCP_ENDPOINT=http://genfeed.localhost:3014/mcp
-NEXT_PUBLIC_API_URL=http://genfeed.localhost:3010/v1
-NEXT_PUBLIC_WS_ENDPOINT=http://genfeed.localhost:3111
+NEXT_PUBLIC_API_ENDPOINT=https://api.genfeed.localhost/v1
+NEXT_PUBLIC_MCP_ENDPOINT=https://mcp.genfeed.localhost/mcp
+NEXT_PUBLIC_API_URL=https://api.genfeed.localhost/v1
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_API_URL key should go before the NEXT_PUBLIC_MCP_ENDPOINT key

(UnorderedKey)